### PR TITLE
Feature/refactor suggested content api

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: modules/custom/moj_resources/src/SuggestedContentApiClass.php [hunk 1]
```diff
@@ -129,88 +128,79 @@ private function getSuggestions($nid, $number, $prison)
...
    */
   private function getTagIds($tags) {
-    $tag_ids = [];
-    $number_tags = count($tags);
+    $tagIds = [];
+    $numberTags = count($tags);
 
-    for ($i = 0; $i < $number_tags; $i++) {
-      array_push($tag_ids, $tags[$i]->target_id);
+    for ($i = 0; $i < $numberTags; $i++) {
```
Comment (LucasCairns) [inline modules/custom/moj_resources/src/SuggestedContentApiClass.php:134]: Could this be an `array_map()`?

**Review by LucasCairns — COMMENTED** [review]
This PR is looking good, loving the naming of the things, certainly improves readability

File: modules/custom/moj_resources/src/SuggestedContentApiClass.php (hunk 1 above)
Comment (psoleckimoj) [reply modules/custom/moj_resources/src/SuggestedContentApiClass.php:134]: @LucasCairns probably, will have a nosey!

**Review by LucasCairns — APPROVED** [review]
👍

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Pull request merged**